doc/source/conf.py

```diff
@@ -23,6 +24,29 @@
 except TypeError:
     pass
 
+from sphinx.application import Sphinx
+
+HERE = Path(__file__).parent
+
+
+def convert_md_to_ipynb(app: Sphinx, *args, **kwargs):
+    import subprocess
+    import sys
+    print("Converting Markdown files to IPyNB...")
+    subprocess.check_call(
+        [
+            sys.executable,
+            "-m",
+            "jupytext",
+            "--to",
+            "ipynb",
+            f"{HERE / 'regression' / '*.md'}",
```

Comment:
Question: do we want the files here to be executed as well when they are being converted (they are not executed right now)? This will take longer for larger projects, but we can benefit from that option in the sense that there are not a lot of notebooks here, and notebooks will have their outputs ready when users view the documentation and open an interactive notebook through the `NotebookLite` directive. There are three other points to consider:

1. Users might like the notebook to be empty so that they can run the cells and see for themselves
2. Large outputs and admonitions from _expected_ errors might make them feel something is wrong on their end until they read through the notebook in depth, which is something we don't want
3. The outputs will get embedded into the HTML page already when MyST-NB will have executed them when building the documentation, so it is not necessary that we need outputs in both places.

I would lean towards (1), personally.

Reply:
> The outputs will get embedded into the HTML page already when MyST-NB will have executed them when building the documentation, so it is not necessary that we need outputs in both places.

As long as this is guaranteed for the generated html docs, I'm happy with the choice for (1).

Reply:
If you mean this for the downloadable notebooks, then yes, they will be empty. The notebooks that MyST-NB generates/caches are placed in `docs/jupyter_execute`, which is not where the download link points to. Resolving this conversation for now.